bridge-ffi/include/sdk-core-bridge.h

```diff
@@ -0,0 +1,131 @@
+#pragma once
+
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+/**
+ * A core instance owned by Core. This must be passed to tmprl_core_shutdown
+ * when no longer in use which will free the resources.
+ */
+typedef struct tmprl_core_t tmprl_core_t;
+
+/**
+ * A runtime owned by Core. This must be passed to tmprl_runtime_free when no
+ * longer in use. This must not be freed until every call to every tmprl_core_t
+ * instance created with this runtime has been shutdown.
+ */
+typedef struct tmprl_runtime_t tmprl_runtime_t;
+
+/**
+ * A set of bytes owned by Core. This must always be passed to
+ * tmprl_bytes_free when no longer in use.
+ */
+typedef struct tmprl_bytes_t {
+  const uint8_t *bytes;
+  size_t len;
+  size_t cap;
+  bool disable_free;
```

Comment:
What's this param for?

Reply:
I figured it wasn't worth creating this struct for empty responses since they happen so often, so I have them as lazy_statics which means I don't want them freed.

Reply:
I'm wondering if we should have a different name for this like `is_static` or at least document what this flag does and when to use it.

Reply:
My next push is saying this is for internal use only

Reply:
Done